binsync/ui/config_dialog.py

```diff
@@ -522,21 +522,24 @@ def load_saved_config(self):
         if not config:
             return None
 
-        if binary_hash not in config.recent_projects.keys():
-            return None
-
-        project_data_dicts = config.recent_projects[binary_hash]
-        confs = []
-        for project_state in project_data_dicts:
-            project_data = ProjectData.get_from_state(project_state)
-            user = project_data.user or ""
-            repo = project_data.repo_path or ""
-            remote = project_data.remote if project_data.remote and not project_data.repo_path else ""
-
-            if not user and not repo:
-                confs.append(None)
-
-            confs.append(f"{repo}:{user}")
+        try:
```

Comment:
Would we be able to reduce the scope of this `try-catch` case? What causes the `AttributeError`?